qa/prscript.py

```diff
@@ -217,7 +217,7 @@ def FindBuild(branch, extension = "", builder_name = None):
 def GetBuildStatus(builder, buildid, extension="", builder_name = None):
     if builder_name == None:
         builder_name = username + extension
-    # https://buildbot.suricata-ids.org/json/builders/build%20deb6/builds/11
+    # https://buildbot.oisf.net/json/builders/build%20deb6/builds/11
```

Comment:
Not sure if this is even used. Couldn't really find the buildbot on any page. I have seen that the GetBuildStatus function is used for building Docker images - this script seems to contain Eric's hard-coded references and the last push to his Docker repo was ~3years ago.
https://hub.docker.com/layers/regit/suri-buildbot/latest/images/sha256-1f8902b075d82e50a3159d1f1388b5eb1f046c04845611a5bba16901856856d4?context=explore

Reply:
@regit I think its time to remove this. Perhaps it could be hosted elsewhere if it is still useful to someone?